lupa/tests/test.py

```diff
@@ -162,7 +162,7 @@ def test_eval_error_message_decoding(self):
         try:
             self.lua.eval('require "UNKNOWNöMODULEäNAME"')
         except self.lupa.LuaError:
-            error = '%s'.decode('ASCII') % sys.exc_info()[1]
+            error = str(sys.exc_info()[1])
```

Comment:
exc_info is already always a `str`

Reply:
No, it's an exception instance. But looking at it again, I noticed that we can avoid calling `sys.exc_info` all together just to get at the current exception that we're handling.

Reply:
I'm happy this goes well. Some other projects want to keep their cargo-culted `compat.py` forever.
There's always opportunity to learn new tricks.